backend/open_webui/routers/groups.py

```
@@ -96,6 +103,25 @@ async def update_group_by_id(
         )
 
 
+############################
+# GetAvailableDomains
+############################
+
+
+@router.get("/domains", response_model=list[str])
+async def get_available_domains(user=Depends(get_admin_user)):
```

Ruff (0.13.2)

Unused function argument: `user`

(ARG001)